packages/tenant-process/src/services/tenantService.ts

```diff
@@ -1153,6 +1156,59 @@ export function tenantServiceBuilder(
       logger.info(`Retrieving Tenant with Selfcare Id ${selfcareId}`);
       return readModelService.getTenantBySelfcareId(selfcareId);
     },
+
+    async addCertifierId(
+      {
+        tenantId,
+        certifierId,
+        correlationId,
+      }: {
+        tenantId: TenantId;
+        certifierId: string;
+        correlationId: string;
+      },
+      logger: Logger
+    ): Promise<Tenant> {
+      logger.info(`Adding certifierId to Tenant ${tenantId}`);
+
+      const tenant = await retrieveTenant(tenantId, readModelService);
+
+      const certifierFeature = tenant.data.features.find(
+        (a) => a.certifierId === certifierId
+      );
+      if (certifierFeature) {
```

Comment:
as per offline discussion, checks should be:
1. if the tenant has already the given `certifierId`, throw an error
2. if the tenant has a different `certifierId`
    - if at least an attribute exists with the tenant `certifierId`, throw an error (different from point 1)
3. else, assign the `certifierId`

Reply:
@rGregnanin I think that what @galales means in the second point is the following: check if that tenant has already assigned a certified attribute to other tenants.

Reply:
That is to say: check in the `attributes` collections if there is at least one attribute with `origin: certifierId`.

Reply:
> That is to say: check in the `attributes` collections if there is at least one attribute with `origin: certifierId`.

this is what I meant (no need to check if the attribute has actually been assigned)

Reply:
Thanks for the clarification